src/injected/web/gm-wrapper.js

```diff
@@ -50,6 +50,7 @@ export function wrapGM(script, code, cache, injectInto) {
     gm.window = unsafeWindow;
   } else {
     thisObj = getWrapper();
+    thisObj.wrappedJSObject = window.wrappedJSObject;
```

Comment:
How about checking the existence of `window.wrappedJSObject` first so that there won't be an undefined property on `wrapper` when it's not in Firefox?

Reply:
I thought about that before and decided it doesn't matter.  Do you think it does?

Reply:
...it's essentially the same as `browser` that we add unnecessarily in Chrome.

Reply:
Yep